app/src/main/java/net/squanchy/analytics/DisabledAnalytics.kt

```diff
@@ -0,0 +1,50 @@
+package net.squanchy.analytics
+
+import android.app.Activity
+import net.squanchy.signin.SignInOrigin
+import net.squanchy.wificonfig.WifiConfigOrigin
+import timber.log.Timber
+
+class DisabledAnalytics : Analytics {
+    override fun initializeStaticUserProperties() = Unit
+
+    override fun trackPageView(activity: Activity, screenName: String, screenClassOverride: String?) = Unit
+
+    override fun trackPageViewOnFirebaseAnalytics(activity: Activity, screenName: String, screenClassOverride: String?) = Unit
+
+    override fun trackPageViewOnCrashlytics(screenName: String) = Unit
+
+    override fun trackItemSelected(contentType: ContentType, itemId: String) = Unit
+
+    override fun trackItemSelectedOnFirebaseAnalytics(contentType: ContentType, itemId: String) = Unit
+
+    override fun trackItemSelectedOnCrashlytics(contentType: ContentType, itemId: String) = Unit
+
+    override fun setupExceptionLogging() {
+        Timber.forest().find {
```

Comment:
Why not:

```kotlin
Timber.forest()
    .find { it is CrashlyticsErrorsTree }
    ?.let { Timber.uproot(it) }
```

So it's consistent with the app code style?

Reply:
Makes sense, will reformat 👍

Reply:
aa21359